test/System/MemoryInputStream.cpp

```diff
@@ -0,0 +1,32 @@
+#include <SFML/System/MemoryInputStream.hpp>
+#include <string_view>
+#include "SystemUtil.hpp"
```

Comment:
One last thing, standard library headers come last by convention. See https://www.sfml-dev.org/style.php#includes
```suggestion
#include <SFML/System/MemoryInputStream.hpp>
#include "SystemUtil.hpp"
#include <string_view>
```

Reply:
I don't even think `SystemUtil.hpp` is getting used.

Reply:
> I don't even think `SystemUtil.hpp` is getting used.

No seems not, I think it's safe to remove it from the includes

Reply:
`<string>` is missing now :smile:
A raw array could also be used instead of the string.

Reply:
It appears it was how this file was getting `std::string` for the Linux GCC build so I'll also add the `<string>` include on